### DISCUSSION TITLE
Is it possible to run prettier format?

### DISCUSSION THREAD

**Discussion by dpyzo0o** [codesandbox/sandpack#285] [General]
We can run prettier to format code in CodeSandbox, is it doable in sandpack?

**Comment by danilowoz** [comment]
Hey! Yes, it's totally possible.
Here's a very basic implementation of how to do it
https://codesandbox.io/s/sandpack-prettier-1po91?file=/src/App.js

Hope it helps you

**Reply by dpyzo0o** [reply]
Thanks!
